### PR TITLE
CTX-5921: Added new condition in onBeforeCommandExecute() callback wh…

### PR DESCRIPTION
**Pull request by bogdant36**
…ich will check if "command_name" is inside excludeOptions also.

### CODE REVIEW

File: coretex/cli/main.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def update() -> None:
 
 
 @click.group(cls = ClickExceptionInterceptor)
-@utils.onBeforeCommandExecute(utils.checkLibVersion)
+@utils.onBeforeCommandExecute(utils.checkLibVersion, excludeOptions = ["update"])
```
Comment (dule1322) [inline coretex/cli/main.py:64]: Will this exclude `coretex node update` as well, or just `coretex update` command?

File: coretex/cli/modules/utils.py [hunk 2]
```diff
@@ -157,6 +157,9 @@ def onBeforeCommandExecute(fun: Callable[..., Any], excludeOptions: Optional[Lis
     def decorator(f: Callable[..., Any]) -> Callable[..., Any]:
         @wraps(f)
         def wrapper(*args: Any, **kwargs: Any) -> Any:
+            if click.get_current_context().invoked_subcommand in excludeOptions:
```
Comment (dule1322) [inline coretex/cli/modules/utils.py:160]: `excludedOptions` is used for excluding parametar of type option when an command is run. Can we add `excludedCommands`, or `excludedSubcommands` parameter to use in cases like these?

**Review by dule1322 — CHANGES_REQUESTED** [review]

File: coretex/cli/main.py (hunk 1 above)
Comment (bogdant36) [reply coretex/cli/main.py:64]: just coretex update command.

Comment (dule1322) [reply coretex/cli/main.py:64]: This is still using `excludeOptions`, shouldn't it use `excludeSubcommands`?

**Review by dule1322 — APPROVED** [review]

**Pull request merged**